Share button

SamWhited opened the pull request:
Add a share button to the action bar as well (branched from #2)

![ss](https://cloud.githubusercontent.com/assets/512573/6546693/1ce7c194-c58b-11e4-9bf0-b9128916ef48.png)

iNPUTmice commented:
Hi,
thank you for your work. I think the share button is a great addition. Will this intent offer to open the geo uri in map-applications as well? Or is this a pure text intent?

One minor detail that might sound a bit weird but can we move the copy-to-clipboard action into the overflow menu (with icon, and `showAsAction="never"`) I feel like this is a secondary option that might have some small use cases but most users will just use the intent because intents are Androids primary way of sharing information (instead of using the clipboard)

SamWhited commented:
> Will this intent offer to open the geo uri in map-applications as well? Or is this a pure text intent?

That's a good point; it's a pure text URL, so unless the map app will search by lat/long, I don't think it will work. It would probably be easy to throw in whatever data map apps are expecting later (or maybe the share widget has that built in somehow?).

> One minor detail that might sound a bit weird but can we move the copy-to-clipboard action into the overflow menu

Yah, I thought something looked weird but I wasn't sure what. I'll do this when I'm next at home at my computer.

SamWhited commented:
Done. Sorry that took me so long for such a tiny change.

SamWhited commented:
I changed the "share" and "copy" buttons to copy a Geo URI, as that's what ends up being sent by the intent anyways, and is probably what will be most useful for eg. launching another map application as previously discussed. Also rebased and squashed.
